lib/iris/fileformats/netcdf.py

```diff
@@ -1538,6 +1538,18 @@ def add_ellipsoid(ellipsoid):
                     cf_var_grid.scale_factor_at_central_meridian = (
                         cs.scale_factor_at_central_meridian)
 
+                # merc
+                elif isinstance(cs, iris.coord_systems.Mercator):
+                    if cs.ellipsoid:
+                        add_ellipsoid(cs.ellipsoid)
+                    cf_var_grid.longitude_of_projection_origin = (
+                        cs.longitude_of_projection_origin)
```

Comment:
However, [pep8](https://www.python.org/dev/peps/pep-0008/) doesn't say anything definitive about this either way, so let's leave it following the style of other parts of this module.